db/migrate/20150802171642_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string  :email
+      t.string  :password
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.